### PR TITLE
RD-168 Update cluster status retrieval

### PR DESCRIPTION
**Pull request by mateumann**

### CODE REVIEW

File: rest-service/manager_rest/cluster_status_manager.py [hunk 1]
```diff
@@ -98,7 +98,7 @@
 
 QUERY_STRINGS = {
     CloudifyNodeType.DB:
-        '(up{job=~".*postgresql"} == 1 and pg_up{job=~".*postgresql"} == 1)',
+        '(pg_up{job=~".*postgresql"} == 1 and up{job=~".*postgresql"} == 1)',
```
Comment (mateumann) [inline rest-service/manager_rest/cluster_status_manager.py:101]: This way we don't report vague `up` metrics, but a more precise `pg_up`.

File: rest-service/manager_rest/cluster_status_manager.py [hunk 2]
```diff
@@ -546,6 +546,7 @@ def _process_metric(metric, timestamp, healthy):
         'healthy': healthy,
         'metric_type': metric.get('__name__', 'unknown'),
         'metric_name': metric_name,
+        'host': metric.get('host', 'localhost')
```
Comment (mateumann) [inline rest-service/manager_rest/cluster_status_manager.py]: Relying on specific format of `metric_name` is probably not the best idea.

File: rest-service/manager_rest/cluster_status_manager.py [hunk 3]
```diff
@@ -428,14 +449,19 @@ def append_service_result(pm, res):
...
             if 'instances' not in service_results[dm]['extra_info'][pm]:
                 service_results[dm]['extra_info'][pm]['instances'] = []
-            # ... append an instance to the list of instances
-            service_results[dm]['extra_info'][pm]['instances'].append(
-                res['extra_info'][pm].get('instances'))
+            # keep only one instance per host
+            for res_instance in res['extra_info'][pm].get('instances'):
+                if res_instance.get('host') not in [
+                        instance.get('host') for instance in
+                        service_results[dm]['extra_info'][pm]['instances']]:
```
Comment (mateumann) [inline rest-service/manager_rest/cluster_status_manager.py:461]: This is madness, but rather a necessary one. We want to be backwards compatible, thus we should keep this extremely deeply nested structure.

File: rest-service/manager_rest/cluster_status_manager.py [hunk 4]
```diff
@@ -517,13 +546,16 @@ def _process_metric(metric, timestamp, healthy):
...
+        metric_name += ' ({})'.format(metric.get('instance'))
+    if metric.get('host'):
+        metric_name += ' on {}'.format(metric.get('host'))
 
     return {
         'last_check': last_check,
         'healthy': healthy,
         'metric_type': metric.get('__name__', 'unknown'),
         'metric_name': metric_name,
+        'host': metric.get('host', 'localhost'),
```
Comment (mateumann) [inline rest-service/manager_rest/cluster_status_manager.py:558]: Relying on specific `metric_name` format is probably not the best idea.

**Review by tehasdf — APPROVED** [review]

**Pull request merged**